python/ray/serve/http_adapters.py

```diff
@@ -0,0 +1,59 @@
+from io import BytesIO
+from typing import List, Optional, Union
+
+from fastapi import File
+from pydantic import BaseModel, Field
+
+from ray.serve.model_wrappers import DataBatchType
+from ray.serve.utils import require_packages
+
+
+_1DArray = List[float]
+_2DArray = List[List[float]]
+_3DArray = List[List[List[float]]]
+_4DArray = List[List[List[List[float]]]]
+
+
+class NdArray(BaseModel):
+    """Schema for numeric array input."""
```

Comment:
is this numpy-specific btw? if so, can we call it that for clarity? `NumpyNdarray`

Reply:
It's not specific to numpy. the input type is just nested list